handlers.go

```diff
@@ -32,12 +32,18 @@ func handleStatusCommand(update tgbotapi.Update) tgbotapi.MessageConfig {
 	} else {
 		currentTime := time.Now()
 		diff := currentTime.Sub(lastWorkout.CreatedAt)
-		days := int(5 - diff.Hours()/24)
-		msg.Text = fmt.Sprintf("%s, your last workout was on %s\nYou have %d days and %d hours left.",
+		timeLeftMessage := ""
+		if diff.Hours() < 120 {
+			days := int(5 - diff.Hours()/24)
```

Comment:
@omerxx  - something is weird here. 
What will it return on 119 hours? On 121 hours?

Reply:
@Amitbergman for anything > 120 the `user.active` would be `false` so he won't be able to speak to the bot anyway...
As for 119, I'd expect it to be 0 days and 1 (or 0?) hours - is this would you're suggesting to fix?

Ideally I'd want this to not even mention "days" if it's 0 (and maybe say "day" if 1) and then make sense of the hours too..

1. What do you think?
2. Did you test it?

Reply:
This is my test user getting banned and then trying to query status : no response (handling >120 is not critical)
![image](https://github.com/omerxx/fatbot/assets/12237955/a8f1a0a4-d9df-4d47-a2b6-54ee8d899a05)

Reply:
@Amitbergman ?